app/src/main/AndroidManifest.xml

```diff
@@ -4,6 +4,7 @@
 
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.INTERNET" /> <!-- New one -->
```

Comment:
ChatGPT leftover ?

Reply:
`I still have my email in the field. But I think we just need to clear the db.`
For this yes, the people already in the db will still have they're address but normally new person won't. They will just have "Unknown Location"
(I changed it in the db for already connected user)

Reply:
> ChatGPT leftover ?

yes some artefact of the map location picker good catch :))

Reply:
`Second thing, we could replace the input text by the proposed address when the field becomes inactive. So that when we click elsewhere than in the box after inputting our address it gets replaced by the exact one
`
For this I will try to find a way but there isn't really a way to know when the textfield is disable, how ever I can make it so when we click on the other text field or when we click on the picture. Could it be enough ?